shard.yml

```diff
@@ -68,3 +68,7 @@ dependencies:
   sqlite3:
     github: crystal-lang/crystal-sqlite3
     version: ~> 0.8.3
+
+  watcher:
+    github: faustinoaq/watcher
```

Comment:
Is this a replacement for sentry? If so, can we remove sentry?

Reply:
I removed both 👍 less dependencies, less to worry about